LiftOff-Project/ClientApp/src/components/NavMenu.js

```diff
@@ -42,7 +42,7 @@ export class NavMenu extends Component {
                 <NavLink tag={Link} className="text-dark" to="/create-account">Create Account</NavLink>
               </NavItem>
               <NavItem>
-                <NavLink tag={Link} className="text-dark" to="/my-watch-list">My Watch List</NavLink>
+                <NavLink tag={Link} className="text-dark" to="/user">Users</NavLink>
```

Comment:
This can move anywhere as long as it doesn't break main's navbar. It's not a huge deal to just take main's navbar design for now.